src/vm/excep.cpp

```diff
@@ -7280,8 +7280,17 @@ bool IsIPInMarkedJitHelper(UINT_PTR uControlPc)
 {
     LIMITED_METHOD_CONTRACT;
 
+#ifdef _ARM_
+    // The least bit denoting THUMB might confuse the range check because
+    // PC values have the least bit reset. It causes a bug if PC is at
+    // the first byte of the function range.
+#define CHECK_RANGE(name) \
+    if ((GetEEFuncEntryPoint(name)|0x1) <= (uControlPc|0x1) && (uControlPc|0x1) < (GetEEFuncEntryPoint(name##_End)|0x1)) return true;
```

Comment:
There are possibilities that we might hit a stack frame with ARM instructions (native stacks). I do not think it is safe to do |0x1 on PC at PAL/ARM/Linux.

I'll do
```
#define CHECK_RANGE(name) \
    if (PCODEToPINSTR(GetEEFuncEntryPoint(name)) <= PCODEToPINSTR(uControlPc) && PCODEToPINSTR(uControlPc) < PCODEToPINSTR(GetEEFuncEntryPoint(name##_End))) return true;
```